tables/hu-backtranslate-word-corrections.cti

```diff
@@ -2183,6 +2192,8 @@ nofor correct "Orosszak" "Oroszszak"	For example Oroszszakértője word
 nofor correct "OROSSZAK" "OROSZSZAK"	For example OROSZSZAKÉRTŐJE word
 nofor correct "amócaön" *	For example szamócaöntettel word
 nofor correct "AMÓCAÖN" *	For example SZAMÓCAÖNTETTEL word
+nofor correct "icaönkén" *	For example cicaönkéntes word
+nofor correct "ICAÖNKÉN" *	For example CICAÖNKÉNTES word
```

Comment:
What is the point of this rule? Doesn't it just copy the input verbatim into the output?

I mean when the action part contains a * and the test part contains no []